test/Base_Tests/src/Semantic/Arrow_Interop_Spec.enso

```diff
@@ -0,0 +1,40 @@
+from Standard.Base import all
+import Standard.Base.Errors.Common.Index_Out_Of_Bounds
+import Standard.Base.Errors.Common.Unsupported_Argument_Types
+import Standard.Base.Errors.Common.Not_Invokable
+import Standard.Base.Errors.Common.No_Such_Method
+
+from Standard.Test import all
+
+foreign arrow new_arrow = """
+    new[Int8]
+
+
+add_specs suite_builder =
+    pending = if Polyglot.is_language_installed "arrow" then Nothing else """
+        Can't run Arrow tests, Arrow is not installed.
+
+    suite_builder.group "Arrow" pending=pending group_builder->
+        group_builder.specify "should allow for creating a new fixed Array in Arrow" <|
+            arrow_array_constructor = new_arrow
+            builder = arrow_array_constructor.new 10
+
+            builder.length . should_equal 10
+            builder.at 1 . should_equal Nothing
+            builder.at 10 . should_fail_with Index_Out_Of_Bounds
+            builder.append 2
+            builder.append 10
+            builder.at 1 . should_equal 10
```

Comment:
The value at index 1 is 10 while few lines above it was `Nothing`. That's OK - I am just not sure why `builder` should have `at` method at all?

Moreover `Meta.type_of builder` is Enso `Array` - e.g. one can do `.map` or `.fold` on the `builder` and in general use it as an array.